### ISSUE TITLE
Convert model to TensorFlow training

### ISSUE THREAD

**Issue by JosephPB** [florpi/proyectito#73]
Convert the training process to TensorFlow for better manipulation of loss function.

**Comment by JosephPB** [comment]
TODO:
1. Add functionality - make loss function a function; add sessions
2. Read in checkpoint files when saved
